docker-library/apache-php-mysql/README.md

```diff
@@ -17,21 +17,20 @@ Ubuntu 16.04 is used as the base image.
 This docker image enables you to:
 
 - run a Apache/PHP/MySQL Environment on **Azure Web App on Linux**;
-- connect your App site to **Azure ClearDB** or the builtin MariaDB;
+- set DATABASE_TYPE with **local** will run built-in MariaDB and phpMyAdmin;
+- connect your App site to **Azure ClearDB** or the built-in MariaDB;
 - manage the build-in MariaDB with the built-in phpMyAdmin;
```

Comment:
merge the second in to this line to make it looking like below:

manage the build-in MariaDB with the built-in phpMyAdmin (You need set DATABASE_TYPE to "LOCAL")